apps/meteor/client/views/marketplace/components/MarketplaceHeader.tsx

```diff
@@ -25,8 +25,11 @@ const MarketplaceHeader = ({ title }: { title: string }): ReactElement | null =>
 		return null;
 	}
 
+	// TODO NBJ: Remove this when enterprise is renamed to premium in the Apps data
+	const handleMarketplacePremiumTitle = title === t('Enterprise') ? t('Premium') : title;
```

Comment:
Better handle this is the component above this one, avoiding match **translated** strings.